lib/tests/streamlit/web/server/routes_test.py

```diff
@@ -166,13 +167,22 @@ def test_parse_url_path_200(self):
         for r in responses:
             assert r.code == 200
 
-    def test_parse_url_path_404(self):
+    def test_most_404_return_default(self):
```

Comment:
what do you mean with "most" here? A better name might be `test_nonexistent_urls_return_default_page`, what do you think?